src/server/controllers/rooms.controller.js

```diff
@@ -0,0 +1,157 @@
+const { v4: uuidv4 } = require('uuid');
+
+const rooms = {};
+
+const create = (ws, data) => {
+  const room = rooms[data.room];
```

Comment:
Consider using a more descriptive variable name for `room` to avoid confusion with the `rooms` object. This can help improve code readability.